wavefront/server/apps/call_processing/call_processing/services/tool_wrapper_service.py

```
@@ -0,0 +1,355 @@
+                else:
+                    return {
+                        'success': False,
+                        'error': f'Unsupported HTTP method: {self.method}',
+                        'status_code': 400,
+                    }
+
+                # Parse response
+                response.raise_for_status()
+                try:
+                    data = response.json()
+                except Exception:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)